pkg/validations/createcluster/validateprovider.go

```diff
@@ -0,0 +1,21 @@
+package createcluster
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/features"
+	"github.com/aws/eks-anywhere/pkg/providers"
+)
+
+func ValidateSupportedProvider(provider providers.Provider) error {
```

Comment:
should we move this to `pkg/provider`?

Reply:
Moved this into the providers pkg as `ValidateActive()`. I created this validation from the `create cluster` cmd but noticed `delete cluster` only checks Cloudstack. Should I make this `ValidateActiveCreate()`?